contracts/HashRegistrarSimplified.sol

```diff
@@ -132,8 +133,11 @@ contract Registrar {
     //   Owned -> Open (releaseDeed)
     function state(bytes32 _hash) constant returns (Mode) {
         var entry = _entries[_hash];
-        if(now < entry.registrationDate) {
-            if(now < entry.registrationDate - revealPeriod) {
+        
+        if(!isAllowed(_hash, now)) {
+            return Mode.Unavailable;
```

Comment:
Yes, fair enough, let's leave it as-is then - although I'd prefer a clearer name than 'Unavailable' (not that one comes to mind).